requirements_test.txt

```diff
@@ -2,3 +2,4 @@ pytest>=4.2.1,<5
 pytest-cov>=2.6.1,<3
 pytest-xdist>=1.26.1,<2
 mypy>=0.670,<1
+codecov~=2.0
```

Comment:
We should probably agree to a single standard and either use `>=,<` or `~=` everywhere. I'm okay with both. I think the only disadvantage of tilde notation is that it's not possible to do something like `lib>=0.1.2,<0.1.5`

Reply:
🆗 I will go with `>=,<` to allow us use `lib>=0.1.2,<0.1.5`